client/src/components/MultiJurisdictionAudit/RoundManagement/generateSheets.ts

```diff
@@ -126,6 +126,14 @@ export const downloadAuditBoardCredentials = (
         20,
         140
       )
+      auditBoardCreds.setTextColor(0, 0, 238)
+      auditBoardCreds.textWithLink(
+        'Or you can click here to go there directly.',
+        20,
+        160,
+        { url: `${window.location.origin}/auditboard/${board.passphrase}` }
+      )
+      auditBoardCreds.setTextColor(0, 0, 0)
```

Comment:
my understanding of this task was to change the existing url text to be a hyperlink, not to add a separate link

Reply:
@jonahkagan Yep, the hope was that that would work, but unfortunately it does not. You cannot create a multi-line hyperlink using jsPDF. The only way would be to create a clickable zone over the top of the link, but since the shape of the link is variable, that would be involved. Adding a clickable link made the most sense to me and is the easiest solution. @mcchilders do you think this is adequate?

Reply:
Ok this is kind of ridiculous but I just went and checked the source of textWithLink and it actually does what you describe - it just makes a clickable zone over the top of the link (https://artskydj.github.io/jsPDF/docs/modules_annotations.js.html#line241). so it might actually be totally reasonable to do that. you could make the width `max(width of the text, page width)` or honestly just use `page width` to make it simpler

Reply:
@jonahkagan The documentation for jsPDF is honestly a pain sometimes, haha. Good job digging further! I was still having issues because for some reason just using the dimensions of the text itself wasn't adequate, with the pdf browser still having an edge around it that was giving the wrong link. Also, having the height be dynamic with the unknown string length possibly wrapping made it interesting. Let me know what you think of my solution.